packages/story-editor/src/components/hotlinkModal/hotlinkModal.js

```diff
@@ -85,8 +86,8 @@ function HotlinkModal({
           ref={inputRef}
           onChange={onChange}
           value={link}
-          hint={errorMsg || description}
-          hasError={Boolean(errorMsg)}
+          hint={defaultErrorMsg || errorMsg || description}
```

Comment:
Two issues:

First, the order should probably reversed here (`errorMsg || defaultErrorMsg || description`)

Second, there is no way to clear the `defaultErrorMsg`.

Example:

1. Try to hotlink image with wrong dimensions
2. See error message about incorrect dimensions
3. Try to hotlink another image that will have CORS issues
4. Still see erros message about dimensions
5. Clear the input field
6. Still see the error message about dimensions

![Screenshot 2022-06-24 at 17 00 39](https://user-images.githubusercontent.com/841956/175562705-be19103a-9af4-4ae2-a67c-60dd14e04e26.png)